sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+# must be unique in a given SonarQube instance
+sonar.projectKey=com.cognifide.qa.bb
+# this is the name and version displayed in the SonarQube UI. Was mandatory prior to SonarQube 6.1.
+sonar.projectName=Bobcat
+sonar.projectVersion=1.4.1-SNAPSHOT
+
+# Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
+# This property is optional if sonar.modules is set.
+sonar.sources=.
+sonar.exclusions=**/target

```

Comment:
`docs` folder should be excluded as well - I can see it being processed in Travis log atm.